config/config.yaml

```
@@ -50,6 +51,18 @@ storage:
     # If null - auto selection.
     update_rate_limit: null
 
+    # Soft limit for number of incoming shard transfers per collection on this node.
+    # Only affects automatic shard transfers for recovery, does not affect user requested transfers.
```

I think the word `soft` might create more confusion than clarification. How about something like this?
```suggestion
    # Limit for number of incoming automatic shard transfers per collection on this node, does not affect user-requested transfers.
```

---

Agreed. Thanks!